cartridges/int_affirm/cartridge/templates/resources/affirm.properties

```diff
@@ -2,9 +2,9 @@ metadata.platform_type=Demandware
 metadata.platform_affirm=22.1.0
 metadata.merchant.name=Test Merchant
 
-affirm.production.url=https://api.affirm.com/api
-affirm.production.js=https://cdn1.affirm.com/js/v2/affirm.js
-affirm.sandbox.url=https://sandbox.affirm.com/api
+affirm.production.url=https://api.global.affirm.com/api
+affirm.production.js=https://cdn1.global.affirm.com/js/v2/affirm.js
+affirm.sandbox.url=https://api.global-sandbox.affirm.com/api
 affirm.sandbox.js=https://cdn1-sandbox.affirm.com/js/v2/affirm.js
```

Comment:
Does AFJS need to be global?

Reply:
According to Sergio, `the expectation for prod sandbox is https://sandbox.affirm.com/js/v2/affirm.js but it looks like cdn1`

Reply:
updated to `sandbox.affirm.com/js/v2/affirm.js` which is the expected value